core/src/main/java/io/appflate/restmock/RequestsVerifier.java

```diff
@@ -41,6 +43,26 @@ public static RequestVerification verifyRequest(Matcher<RecordedRequest> matcher
         return new RequestVerification(matcher);
     }
 
+    public static RequestVerification verifyDELETERequest(Matcher<RecordedRequest> matcher) {
+        return verifyRequest(AllOf.allOf(RequestMatchers.isDELETE(), matcher));
+    }
+
+    public static RequestVerification verifyGETRequest(Matcher<RecordedRequest> matcher) {
+        return verifyRequest(AllOf.allOf(RequestMatchers.isGET(), matcher));
+    }
+
+    public static RequestVerification verifyPATCHRequest(Matcher<RecordedRequest> matcher) {
```

Comment:
Could we rename those methods to match `RestMock.when*METHOD*`? So it would be `verifyPATCH, verifyGET` etc.